cmd/update.go

```diff
@@ -135,6 +135,10 @@ func retrieveDNS(c Config) (string, error) {
 		return "", fmt.Errorf("error retrieving DNS: %s", result.Status)
 	}
 
+	if len(result.Records) == 0 {
```

Comment:
I think placing this block before the `if result.Status == "ERROR"` will be better, otherwise, if there's an Error and no records were found, the first conditional won't be too specific in the error message on why that happened

Reply:
That could be misleading though if there's a different error. Records will most likely always be zero when there is an error.

Check your email btw.

Reply:
you're actually right, the error verification should precede the DNS records length check. It's alright then